prboom2/src/SDL/i_video.c

```diff
@@ -1412,6 +1409,9 @@ void I_UpdateVideoMode(void)
     gld_Init(SCREENWIDTH, SCREENHEIGHT);
   }
 
+  ST_SetResolution();
```

Comment:
Why did this move? Was there a bug?

Reply:
Yeah, these can end up making drawing calls before GL is initiated.  In my lightmode branch it caused attempts to use shaders before they are allocated/loaded.